### PR TITLE
helm: relax alerting criteria for CephNodeNetworkPacketDrops

### PR DESCRIPTION
**Pull request by ahgraber**
The alerting criteria are too strict, leading many to silence the alert (boy-who-cried-wolf syndrome) 
and therefore losing any alerting that may be actually relevant to cluster health

Signed-off-by: ahgraber <ahgraber@ninerealmlabs.com>

**Description of your changes:**

This change relaxes the alerting criteria for `CephNodeNetworkPacketDrops`:
* from drops > 0.01% to drops > 0.5%
* from >= 10 drops to >= 75 drops
* adds `for 1m` to only alert with consistent packet drops over a period of time

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [x] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure)).
- [ ] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [x] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [x] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

File: PendingReleaseNotes.md [hunk 1]
```diff
@@ -11,3 +11,4 @@
 - Change `pspEnable` default value to `false` in helm charts.
 - [Bucket notifications and topics](https://rook.io/docs/rook/latest/Storage-Configuration/Object-Storage-RGW/ceph-object-bucket-notifications/)
   for object stores moved to stable from experimental.
+- Update `CephNodeNetworkPacketDrops` prometheus rule with relaxed alerting criteria
```
Comment (travisn) [inline PendingReleaseNotes.md:14]: These release notes are only for 1.11 feature. We can consider this as a fix to backport to 1.10, so I'd suggest reverting this file.

**Review by travisn — CHANGES_REQUESTED** [review]
- For the commit title, use `monitoring:` instead of helm.
- Please squash to a single commit
- We try to mirror the alerts recommended by the ceph team. Would you consider opening the PR in the ceph repo against https://github.com/ceph/ceph/blob/main/monitoring/ceph-mixin/prometheus_alerts.yml to get feedback there before we merge this? 
- Instead of this PR for all users, consider customizing your alerts as described [here](https://rook.io/docs/rook/latest/Storage-Configuration/Monitoring/ceph-monitoring/#customize-alerts).

**Comment by ahgraber** [comment]
Thanks for the feedback, I'll implement later this week.
I do personally patch the changes according to the documentation as linked; however, when I shared this patch with many in one of my discord communities (kubernetes@home), I had many requests to PR this upstream so folks would stop silencing the alert outright.

**Comment by ahgraber** [comment]
Interestingly, it looks like upstream ceph/ceph  has already implemented part of this by increasing the % threshold and switching from `increase` to `rate`:
https://github.com/ceph/ceph/commit/351e1ac63950164ea5f08a6bfc7c14af586bb208 

I'm happy to close this PR in favor of the linked upstream changes.

Apologies for the commit messiness; I'm unused to doing anything more than simple commits & pulls.

**Comment by galexrt** [comment]
@travisn I think it is reasonable to update the alert rules files in general, seems that there were some changes on Dec 20 2022, see https://github.com/ceph/ceph/blob/main/monitoring/ceph-mixin/prometheus_alerts.yml

@ahgraber Could you update both files with the latest version from the ceph-mixin repo? If you need help with git, feel free to ping me on the [Rook Slack](https://slack.rook.io/).

**Review by travisn — CHANGES_REQUESTED** [review]
@ahgraber It seems there is a problem with the commit history. We just want to ensure only the needed commit is in the PR. Please look at [rebasing on upstream master](https://rook.io/docs/rook/latest/Contributing/development-flow/#updating-your-fork) to update the PR.

**Comment by travisn** [comment]
@ahgraber How about opening a new PR with the single commit? It will work better if you open the PR from a branch in your fork, so we don't have the merge commits getting pulled in.

**Pull request closed without merging**